### ISSUE TITLE
Clarity in documentation (Win11)

### ISSUE THREAD

**Issue by bugsbunny88** [h2oai/h2ogpt#453]
Hello,
great project and just posting this so its some help , which I can probably provide some insight to improve the docs. 

When a user enters the tldr ; in my case on a laptop I end up here quickly (https://github.com/h2oai/h2ogpt/blob/main/docs/README_WINDOWS.md) ; but from here one gets lost on the following aspects . 

1) we do Run: wsl --install ; is this line meaning that h20gpt will not work on win11 ? that the conda environment etc must all be done through wsl ? 

2) conda install cudatoolkit -c conda-forge -y  # required for bitsandbytes
- its not clear to me why bitsandbytes would or would not be required 

3) at # (9) the decision point to go for CPU or GPU is not clear ; because for instance some helpful criteria would be appreciated. In my scenario I am on a laptop with 16gb ram, 3070ti with 8gb vram, and a 5900hx cpu .... If I have an idea of what criteria would be a sensible default I might go one way or the other

4) I don't understand what compute <7 means here : "unless you have compute capability <7.0, then your GPU only supports 8-bit (not 4-bit) and you should install older bitsandbytes:" 

5) for # 11 ; its not clear if these are being installed just on windows locally or within conda , and if within environment then how... "For supporting Word and Excel documents download and install libreoffice: https://www.libreoffice.org/download/download-libreoffice/ . To support OCR, downnload and install tesseract."

6) how does autogptq interact with bitsandbytes ... its written optional... but completely no idea which decision point I need to make "Install optional AutoGPTQ dependency:"

I would just like to say, this project is the best documented. and an excellent resource and probably the end-all-be-all. So I seek your guidance in terms of best-practices ; but also understanding.

**Comment by a3nima** [comment]
My answers, to my best knowledge, not a specialist in this ;) 
1. According to the readme win10+11 install is identical. No wsl. https://github.com/h2oai/h2ogpt/blob/main/docs/README_WINDOWS.md
2. dunno
3. With your GPU you will use that at least with models that fit into your VRAM.
4. older GPUs dont have this..
5. when installed there should be an automation possible, where h2ogpt does the conversion automatically, not working for my libre office installation at first try (reported).
6. did not work for me.. maybe some models need it. (reported)

**Comment by pseudotensor** [comment]
Hi, sorry for any confusions.  

1. I'm actually not sure wsl is required.  I originally borrowed the instructions from PrivateGPT, but then went through on my windows 10 box and added many more details.  I felt like like wsl wasn't really required, but I had no issues on my windows 10 box, so I didn't worry about it.  If you guys some insights into that would be good.  Should it be removed?  does it install libraries etc. that are required?  It's hard in windows because once I've installed stuff I can't easily uninstall, and there is no clean docker AFAIK.

2. I tried to clarify this in new PR

3. Yes, in general the only help we provide is in the main readme.md under supported hardware.  But a table there would be nice.

4. jllllll make a PR and updated bitsandbytes there and removed that part.

5. I updated the readme a bit, I think windows/mac works without libreoffice as long as one has word/excel installed.  Those packages have their own long steps, and I added tesseract link to windows instructions.

6. Yes, someone also pointed out in discord.  I need to try windows a bit later and see what is wrong.  Any insights you guys have is iseful.
